frontend/src/pages/Dashboard/page.module.css

```diff
@@ -14,8 +14,15 @@
   display: flex;
   justify-content: space-between;
   align-items: center;
+  height: 150px;
 }
 
-.margin3 {
-  margin: 3rem;
+.rightSection {
+  margin-right: 3rem;
+  display: flex;
+  align-items: center;
+}
+
+.rightSection > div {
+  margin-left: 1rem;
 }
```

Comment:
sorry if this wasn't clear, i wanted you to follow the header styles that are used for the employee and students pages in the designs
![image](https://user-images.githubusercontent.com/20343975/113207524-b3f90080-923e-11eb-94b5-bda0f86d9331.png)
if this changes the homepage styles thats okay, that page will require some more work in a different PR

Reply:
do you mean I should use the existing `.margin3` css class instead of introducing new `.rightSection` css class?

if I use `.margin3`, the notification bell component doesn't render at the correct position. Would that be okay?

![image](https://user-images.githubusercontent.com/32003815/113223017-388d6280-92c3-11eb-8bbe-528e080fb8c3.png)

Reply:
nah you can use the rightSection class. feel free to introduce new styles, change existing styles, and reformat any pages as you see fit, as long as all the pages align with the designs (besides homepage, but it would be good if you could structure it similar to other pages if possible)

Reply:
okay sounds good. which part did you feel were different from the designs?

Reply:
mainly the padding and header font size

Reply:
okay I've edited the css so that the padding and header font size are closer to the designs. let me know if there's anything else that you want fixed